Cargo.toml

```diff
@@ -35,7 +35,8 @@ serde = "0.8"
 serde_derive = { version = "0.8", optional = true }
 serde_json = "0.8"
 lazy_static = "0.2.1"
-ssdp = { version = "0.4", optional = true }
+ssdp = { version = "0.5", optional = true }
```

Comment:
It is in fact. ssdp 0.5 has merged a pull-request by me that integrates error-chain as well.